### ISSUE TITLE
Designing Shift registers using Sequential Circuits

### ISSUE THREAD

**Issue by eKuG** [CircuitVerse/CircuitVerse#523]
Shift Register is a group of flip flops used to store multiple bits of data. The bits stored in such registers can be made to move within the registers and in/out of the registers by applying clock pulses.

Use https://circuitverse.org/simulator

I recommend making it a standard Example.

**Comment by eKuG** [comment]
Designing a Universal Shift Register would give a proper understanding for every type.

**Comment by Ananthavijay** [comment]
@eKuG

**Comment by Ananthavijay** [comment]
@eKuG I have made a universal shift register. Check it out here : https://circuitverse.org/users/13891/projects/45314 so that I can add it into the example list as suggested

**Comment by eKuG** [comment]
Yes you may use it. It would be better if you could explain the working for other people. Also, check if it is in the docs or not. And add it there as well

**Comment by ciphereck** [comment]
@Ananthavijay are you still working on this? I see you closed your pull request?

**Comment by Ananthavijay** [comment]
I delted the branch by mistake, will reopen it

**Comment by Ananthavijay** [comment]
Please reopen this issue
